<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWordFinder</class>
 <widget class="QDialog" name="CWordFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="bbxButtons">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>128</y>
     <width>389</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cboWordTerm">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>8</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cboWordValue">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>36</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkWordIsLearned">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>68</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Is learned:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkWordIsMarked">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>96</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Is marked:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>12</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Term:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>40</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
